static/sass/global.scss

```diff
@@ -1,3 +1,380 @@
+        &:before {
+          background: lighten($color-brand, 12%);
+          content: ' ';
+          display: block;
+          height: 100%;
+          left: 0;
+          top: 0;
+          position: absolute;
+          width: 1px;
+        }  
+      }    
+    }
+  }
+
+  .p-navigation .p-navigation__logo {
+    border: 0;
+    padding-top: 6px;
+    padding-right: 5px;
+    
+    .p-navigation__link {
+      background: none;
+      border: 0;
+      color: $color-x-light;
+      font-size: 1.4375rem;
+      font-weight: normal;
+      padding: 0;
+      width: 100%;
+      
+      &:before {
+        width: 0;
+      }
+      
+      &:last-of-type {
+        border: 0;
+      }
+    }
+  }
+  
+  .p-navigation .p-navigation__logo .p-navigation__link:hover {
+    background: none;
+    text-decoration: none;
+  }
+  
+  .p-navigation .p-navigation__link:hover {
+    background: #f8f8f8;
+    border-color: #cdcdcd;
+    text-decoration: none;
+    
+    @media only screen and (min-width: $breakpoint-medium) {
+      background: lighten($color-brand, 6%);
+      border-color: darken($color-brand, 6%);
+    }
+  }
+}
+
+.theme__aside {
+  margin-top: -.5rem;
+  padding-top: 0;
+}
+
+.theme > .p-footer {
+  border-color: $color-mid-light;
+}
+
+.p-inline-list--tags {
+  margin-top: -1rem;
+
+  &__link {
+    color: $color-mid-dark !important;
+    
+    &:link,
+    &:visited {
+      color: $color-mid-dark;    
+      text-decoration: none;
+    }
+  }
+}
+
+.p-toc {
+  border-left-color: $color-mid-light;  
+  padding-top: .75rem;
+  
+  &__header {
+    color: $color-mid-dark;
+  }
+  
+  &__item {
+    margin-bottom: .5rem;
+  }
+  
+  &__link,
+  &__link:visited,
+  &__link:link {    
+    color: $color-mid-dark;
+  }
+}
+
+// pagination
+.p-pagination__item {
+  float: left;
+}
+
+.p-pagination__item--left {
+  float: right;
+}
+
+.pagination__link--previous {
+  float: left;
+  margin-bottom: 1rem;
+}
+ 
+.p-pagination__link {
+  &:link,
+  &:visited {
+    border: 1px solid $color-mid-dark;
+    color: inherit;
+  }
+}
+
+// cards
+.row--cards {
+  &__title {
+    background: rgba(0, 0, 0, 0) url("https://assets.ubuntu.com/v1/9334aabe-external-link-cool-grey.svg") no-repeat scroll left center / 16px auto;
+    color: $color-mid-dark;
+    font-size: 1rem;
+    margin-bottom: .75rem;
+    padding-left: 22px;
+    text-transform: uppercase; 
+  }
+  
+  .p-card--highlighted {
+    margin-bottom: 40px;
+    
+    @media only screen and (min-width: $breakpoint-medium) {
+      min-height: 21.5em;
+    }
+    
+    padding-bottom: 40px;
+    position: relative;
+    
+    &__title {
+      font-size: 1.3125rem;
+    }
+    
+    &__source {
+      bottom: 20px;
+      color: $color-mid-dark;
+      margin-bottom: 0;
+      position: absolute;
+      text-transform: uppercase; 
+    }
+
+    &__date {
+      color: $color-mid-dark;
+      margin-top: -1rem;
+    }
+  }
+}
+
+$grid-border-style: 1px dotted $color-mid-light;
```

Comment:
Can you move this variable into the `.grid-list` component to scope it to that component please.